horizons/util/python/decorators.py

```diff
@@ -97,12 +97,11 @@ def __call__(self, *args, **kwargs):
 					del self.cache_dates[key]
 					return self(*args, **kwargs)
 			else:
-				self.cache_dates[key] = time.time() # new entry
+				self.cache_dates[key] = time.time()  # new entry
```

Comment:
Actually, we recommend PEP-8 as style guide, but we don't enforce it (that's why I opened #2482). At least not according to the Wiki. And I'm pretty sure we've had the discussion about inline comments before.

While we're at it, allow me me to quote PEP-8 too:

> A style guide is about consistency. Consistency with this style guide is important. Consistency within a project is more important. Consistency within one module or function is the most important.

https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

You're arguing for **Consistency with this style guide is important**, whereas I actually looked at the rest of the codebase to see that (apparantly) most inline comments use one space. You keep ignoring that statement.